src/PROPOSAL/detail/PROPOSAL/secondaries/parametrization/photoeffect/NaivPhotoeffect.cxx

```diff
@@ -0,0 +1,20 @@
+#include "PROPOSAL/secondaries/parametrization/photoeffect/NaivPhotoeffect.h"
+#include "PROPOSAL/Constants.h"
+
+#include <cmath>
+
+using namespace PROPOSAL;
+
+std::vector<ParticleState> secondaries::NaivPhotoeffect::CalculateSecondaries(
+        StochasticLoss loss, const Component& comp, std::vector<double>&)
+{
+    double I = pow(comp.GetNucCharge() * ALPHA, 2) * ME / 2; // subtract ionization energy of K-shell electron
+    double E = ME + loss.parent_particle_energy - I;
+
+    auto sec = std::vector<ParticleState>();
+    // TODO: Do we want to do something better here?
+    // TODO: Do we need to make sure that the created electron has a valid energy that is above its rest mass? Or is this already secured?
```

Comment:
This should be done, before when calculating the loss.

Reply:
The final state electron should always have an energy above its rest mass, because the final electron should have its rest mass energy plus the photon energy, and the photon energy cannot be negative.